Improve CONTRIBUTING.md with a table of ways to contribute (#22724)
prestodb/presto: CONTRIBUTING.md

# Contributing to Presto

Thanks for your interest in Presto. Our goal is to build a fast, scalable, and reliable distributed SQL query engine for running low latency interactive and batch analytic queries against data sources of all sizes ranging from gigabytes to petabytes.

# What Would You Like to Do?

| Area             | Information                                                                                                                                                                                                                                                                                                                                    |
|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | Getting Started  | 1. Build presto <br/>2. Look for [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tickets. <br/> 3. Reference project boards for open work: https://github.com/prestodb/presto/projects?query=is%3Aopen                                                                                                                                                                    |

Reviewer: ```suggestion
| Getting Started  | 1. Build presto <br/>2. Look for [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tickets. <br/> 3. [Reference project boards for open work](https://github.com/prestodb/presto/projects?query=is%3Aopen).                                                                                                                                              |
```

> | Getting Started  | 1. Build presto <br/>2. Look for [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tickets. <br/> 3. Reference project boards for open work: https://github.com/prestodb/presto/projects?query=is%3Aopen                                                                                                                                                                    |

Reviewer 2: ```suggestion
| Getting Started  | 1. [Build Presto](#building-presto) <br/>2. Look for [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tickets. <br/> 3. Reference project boards for open work: https://github.com/prestodb/presto/projects?query=is%3Aopen                                                                                                                                                                    |
```
Nit: capitalize Presto, add link to the heading.

> | Getting Started  | Read Presto's [open issues](https://github.com/prestodb/presto/issues). Tag issues that would make a good first pull request for new contributors with a [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tag. An easy way to start helping the project is to [open an issue](https://github.com/prestodb/presto/issues/new/choose). Issues can include bugs, new features, or outdated documentation.  |

Reviewer: Can we make getting started be:

1) Build presto (remove section below, inline here).  Look for good first issue tickets.  Reference project boards for open work: https://github.com/prestodb/presto/projects?query=is%3Aopen

| Building Presto  | Please review the minimum system specs [requirements](#requirements) before [building](#buildingpresto) Presto.                                                                                                                                                                                                                                                                    |
| Report Bug       | To report a bug, visit Presto's [open issues](https://github.com/prestodb/presto/issues).                                                                                                                                                                                                                                                                                          |

> | Contributions    | Please read the [contributions](#contributions) section to learn about how you can contribute to Presto, including the submission process, minimum expectations, and guidelines for designing your code.  Ready to open a pull request? Be sure to review the [Pull Request guidelines](#pullrequests).                                                                                                                                                                                                                                                                     || Contributor License Agreement ("CLA") | First-time contributors must sign a CLA. For more information see [Contributor License Agreement ("CLA")](#cla).                                                                                                                                                                                                                                                                                                |

Author: Good catch, thanks!

# Contributing to Presto

> Thanks for your interest in Presto. Our goal is to build a fast, scalable and reliable distributed SQL query engine for running low latency interactive and batch analytic queries against data sources of all sizes ranging from gigabytes to petabytes.

Reviewer 3: Oxford comma after scalable

| Building Presto                       | Please review the minimum system specs [requirements](#requirements) before [building](#buildingpresto) Presto.                                                                                                                                                                                                                                                                    |
| Report Bug                            | To report a bug, visit Presto's [open issues](https://github.com/prestodb/presto/issues).                                                                                                                                                                                                                                                                                          |

> | Contributions                         | Please read the [contributions](#contributions) section to learn about how you can contribute to Presto, including the submission process, minimum expectations, and guidelines for designing your code.                                                                                                                               |
> | Pull Requests                         | Ready to commit your code to the main branch? Be sure to review the [Pull Request guidelines](#pullrequests) .                                                                                                                                                                                                                                                                     |

Reviewer: ```suggestion
| Contributions                         | Please read the [contributions](#contributions) section to learn about how you can contribute to Presto, including the submission process, minimum expectations, and guidelines for designing your code.  Ready to open a pull request? Be sure to review the [Pull Request guidelines](#pullrequests) .                                                                                                                                                                                                                                                                     |
```

> | Contributor License Agreement ("CLA") | For first-time contributors, a CLA must be signed. Read more about it [here](#cla).                                                                                                                                                                                                                                                                                                |

Reviewer 3: active voice: First-time contributors must sign a CLA.

> | Contributor License Agreement ("CLA") | For first-time contributors, a CLA must be signed. Read more about it [here](#cla).                                                                                                                                                                                                                                                                                                |

Reviewer 2: ```suggestion
| Contributor License Agreement ("CLA") | First-time contributors must sign a CLA. For more information see [Contributor License Agreement ("CLA")](#cla).                                                                                                                                                                                                                                                                                                |
```
Avoid "here" in links. For context, see the GitLab Documentation Style Guide entry [Descriptive text rather than here](https://docs.gitlab.com/ee/development/documentation/styleguide/#descriptive-text-rather-than-here).

## Getting Started

> Read Presto's [open issues](https://github.com/prestodb/presto/issues). Tag issues that would make a good first pull request for new contributors with a [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tag. An easy way to start helping the project is to [open an issue](https://github.com/prestodb/presto/issues/new/choose). Issues can include bugs, new features, or outdated documentation. 

Author: Thanks for the feed back. This section is identical to the `Getting Started` cell, so I made the same revision here.
